### PR TITLE
fix(LoginPage): improve submit button UX

### PR DESCRIPTION
**Pull request by Laviro**
As part of the [LoginPage implementation in Foreman](https://github.com/theforeman/foreman/pull/6234) 
Adding loading spinner and disable mode to the login submit button.

### CODE REVIEW

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardForm.js [hunk 1]
```diff
@@ -17,24 +18,18 @@ const LoginCardForm = ({
   rememberMe,
   submitError,
   showError,
-  attributes
+  attributes,
+  isLoading
```
Comment (ohadlevy) [inline packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardForm.js]: perhaps isSubmitting ?

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import PropTypes from 'prop-types';
+import { Button, Icon } from '../../../../index';
+
+const LoginCardSubmitButton = ({ isDisabled, children, isLoading }) => (
+  <Button className="login-pf-submit-button" type="submit" bsStyle="primary" bsSize="large" block disabled={isDisabled}>
+    {children}
+    {isLoading ? (
+      <span>
+        {' '}
+        <Icon name="spinner" spin size="lg" />
```
Comment (ohadlevy) [inline packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js]: perhaps use the spinner component?

Comment (Laviro) [reply packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js]: Do we have one ? :O

**Comment by Laviro** [comment]
Thanks, I will polish the proptypes in another PR,
I will add test later today

**Comment by Laviro** [comment]
@jeff-phillips-18 though it seems that `LoginCardSubmitButton` is covered 100%:
selection_084

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js (hunk 2 above)
Comment (jgiardino) [reply packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js]: Here's a link to the [PF3 react spinner](https://944-pr-patternfly-react-patternfly.surge.sh/patternfly-3/index.html?knob-Loading=true&knob-Size=sm&selectedKind=patternfly-react%2FWidgets%2FSpinner&selectedStory=Spinner&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) (using your storybook preview url, btw)

**Comment by jgiardino** [comment]
I took a quick look, and don't see any issues from the html/css side.

**Pull request merged**